.vuepress/public/_redirects

```diff
@@ -81,11 +81,13 @@
 /reference/features/faceted_search.html              /reference/features/filtering_and_faceted_search.html
 
 # Move under the hood articles to /learn
-/reference/under_the_hood/bucket_sort           /learn/advanced/bucket_sort
 /reference/under_the_hood/concat                /learn/advanced/concat
 /reference/under_the_hood/datatypes             /learn/advanced/datatypes
 /reference/under_the_hood/indexation            /learn/advanced/indexation
 /reference/under_the_hood/prefix                /learn/advanced/prefix
 /reference/under_the_hood/storage               /learn/advanced/storage
 /reference/under_the_hood/tokenization          /learn/advanced/tokenization
-/reference/under_the_hood/typotolerance         /learn/advanced/typotolerance
\ No newline at end of file
+/reference/under_the_hood/typotolerance         /learn/advanced/typotolerance
+
+# Bucket sort
+/reference/under_the_hood/bucket_sort           /learn/core_concepts/relevancy#behavior
```

Comment:
I think you may need to add `.html` for these

Reply:
yeap, you're right 🎀